src/util/piped_process.cpp

```diff
@@ -156,7 +156,7 @@ piped_processt::piped_processt(const std::vector<std::string> &commandvec)
     throw system_exceptiont("Input pipe creation failed for child_std_IN_Rd");
   }
   // Connect to the other side of the pipe
-  child_std_IN_Wr = CreateFileA(
+  child_std_IN_Wr = CreateFile(
```

Comment:
So what if the build configuration (i.e., the one used by GitHub) doesn't match that of the user?

Reply:
The `CreateFile` macro in the windows headers are defined such that it will be resolved to either `CreateFileA` or `CreateFileW`. Both variations are available in all versions of Windows from Windows XP onward  - [Detailed Wikipedia page](https://en.wikipedia.org/wiki/Unicode_in_Microsoft_Windows)

If a string is written to based on one encoding and then reinterpreted as being a string of an entirely different encoding then the result may/will be corrupt. The exact nature and extent of this corruption depends on the 2 encodings concerned and the characters in the string. This can vary between the extremes of no corruption and a string which does not even contain the same number of characters. There is a wide enough set of possibilities than I don't want to go into all of them in detail in this github conversion, if I can avoid it. Note that when reading into these issues there is a distinction between characters/glyphs and their encodings. Where-by every character/glyph can be encoded in multiple different ways which can be varying numbers of bits/bytes wide. I think the important thing is that the API calls are consistent within the CBMC binary. Suffice to say this is an issue of the byte/binary level encoding of characters, not an issue of user locale settings.

This change to the `CreateFile` call makes it consistent with the call to `CreateNamedPipe`. Which also has `CreateNamedPipeA` and `CreateNamedPipeW` variants in the Windows headers.